### PR TITLE
Adding entries for PowerNotifications WinMD file

### PR DESCRIPTION
**Pull request by huzaifa-d**
Adding entries for the new WinMD file, Microsoft.Windows.System.Power.winmd

### CODE REVIEW

**Comment by kythant** [comment]
I've gone and inspected the transprot package to look for the winmd and I didn't see it.
I suspect the line where you have added your code in CopyFilesToStagingDir.ps1 is for publishing to build artifacts. 
 
For the actually Nuget transport package, you'll have to add it at the section for "# WinMD for UWP apps" for this to work.

**Review by kythant — CHANGES_REQUESTED** [review]
I've gone and inspected the transport package to look for the winmd and I didn't see it.
I suspect the line where you have added your code in CopyFilesToStagingDir.ps1 is for publishing to build artifacts.

For publishing to the Nuget transport package, you'll have to add your code at the section for "# WinMD for UWP apps" for this to work.

File: build/NuSpecs/WindowsAppSDK-Nuget-Native.WinRt.props [hunk 1]
```diff
@@ -24,6 +24,11 @@
       <HintPath>$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.Windows.System.winmd</HintPath>
       <Implementation Condition="'$(WindowsAppSDKFrameworkPackage)' != 'true'">$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(_WindowsAppSDKFoundationPlatform)\native\Microsoft.WindowsAppSDK.dll</Implementation>
       <IsWinMDFile>true</IsWinMDFile>
+    </Reference>
+	<Reference Include="Microsoft.Windows.System.Power.winmd">
+      <HintPath>$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.Windows.System.Power.winmd</HintPath>
```
Comment (kythant) [inline build/NuSpecs/WindowsAppSDK-Nuget-Native.WinRt.props:29]: Can this target be on uap10.0?

I've been working on a change to move these winmds files to uap10.0
https://github.com/microsoft/WindowsAppSDK/pull/1219

File: build/NuSpecs/WindowsAppSDK-Nuget-Native.targets [hunk 2]
```diff
@@ -30,6 +30,13 @@
     </Reference>
   </ItemGroup>
 
+    <ItemGroup>
+    <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.Windows.System.Power.winmd">
```
Comment (kythant) [inline build/NuSpecs/WindowsAppSDK-Nuget-Native.targets:34]: Same here uap10.0

File: build/NuSpecs/WindowsAppSDK-Nuget-Native.WinRt.props (hunk 1 above)
Comment (huzaifa-d) [reply build/NuSpecs/WindowsAppSDK-Nuget-Native.WinRt.props]: Done

**Review by kythant — APPROVED** [review]

**Review by DrusTheAxe — APPROVED** [review]

**Pull request merged**